as/plugins/org.jboss.ide.eclipse.as.core/jbosscore/org/jboss/tools/as/core/server/controllable/subsystems/internal/StandardFileSystemPublishController.java

```diff
@@ -408,14 +407,6 @@ public int publishModule(int kind,
 		return acc == null || acc.isOK() ? IServer.PUBLISH_STATE_NONE : IServer.PUBLISH_STATE_UNKNOWN;
 	}
 	
-	protected boolean treatAsBinaryModule(IModule[] module) {
-		return true; // isBinaryModule(module);		
```

Comment:
Rejecting this PR  (no offense) but changing this line back to return isBinaryModule(module).  The reason here is I want to allow a chance to fix the openshift adapter without further changes to astools. Not sure if it will happen, but, give a try.